client/my-sites/site-settings/featured-image-toggle.jsx

```diff
@@ -24,7 +25,14 @@ const FeaturedImageTemplateToggle = ( props ) => {
 				/>
 				<FormSettingExplanation>
 					{ translate(
-						"Includes your post's featured image in the email sent out to your readers."
+						"Includes your post's featured image in the email sent out to your readers. {{link}}Learn more about the featured image{{/link}}.",
```

Comment:
```suggestion
						"Includes your post's featured image in the email sent out to your subscribers. {{link}}Learn more about featured image{{/link}}.",
```
Some copy suggestions:
- Perhaps using subscribers instead of readers would be more coherent with the usage of subscribe around subscribe block, newsletter subscription, etc?
- Removing "the" in "Learn more about ~the~ featured image."

Reply:
Thank you for your feedback, Yan!

I was using the copy suggested by @saygunnyc in pdtkmj-Gy-p2#comment-1149. Let's see what he thinks about the proposed alternative.